project/cmake/scripts/linux/ArchSetup.cmake

```diff
@@ -28,5 +28,4 @@ if(CMAKE_BUILD_TYPE STREQUAL Coverage)
   set(COVERAGE_TEST_BINARY ${APP_NAME_LC}-test)
   set(COVERAGE_SOURCE_DIR ${CORE_SOURCE_DIR})
   set(COVERAGE_EXCLUDES */test/* lib/* */lib/*)
-  include(${CMAKE_CURRENT_LIST_DIR}/CodeCoverage.cmake)
```

Comment:
Thanks! Good idea to move this to the main file. Any reason why you didn't move the whole block?